package.json

```diff
@@ -36,21 +37,53 @@
       ],
       "configuration": "./language-configuration.json"
     }],
-    "configurationDefaults": {
-      "[lc2k]": {
-        "editor.insertSpaces": false,
-        "editor.tabSize": 8,
-        "editor.autoIndent": false,
-        "editor.formatOnSave": false
-      }
-    },
+
     "grammars": [{
       "language": "lc2k",
       "scopeName": "source.lc2k",
       "path": "./syntaxes/lc2k.tmLanguage.json"
     }],
     "capabilities": {
       "definitionProvider": "true"
+    },
+    "configuration": {
+      "type": "object",
+      "title": "LC2K configuration",
+      "properties": {
+        "lc2k.format.enable": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enables this extention to format .as files."
+        },
+        "lc2k.format.addNewlineAtEOF": {
+          "type": "boolean",
+          "default": true,
+          "description": "Add a newline at the end of a file if there isn't one."
+        },
+        "lc2k.format.filloffset": {
+          "type": "integer",
+          "default": 2,
+          "description": "Offset for comments after a .fill, inline is 3."
```

Comment:
What does "inline" mean? Why does inline not match with the default? What is this indentation relative to? The beginning, or extra tabs since the last text. Could be more clear.